trace-sdk/src/test/java/io/bitrise/trace/network/MetricSenderTest.java

```diff
@@ -87,4 +94,60 @@ public void needReschedule_shouldBeTrue_hasAll() {
 
         assertTrue(metricSender.isRescheduleNeeded());
     }
+
+    private final String headerAcceptedMetricsCount = "accepted-metrics-count";
```

Comment:
Maybe we could put these two variables in a place where it is accessible for this class, so we do not have to type them twice, and modify them twice if it changes.

Reply:
Have added the constants to the NetworkCommunicator because that's where the network calls are declared, so it made sense to put the header constants near there. Does that sound okay?

Reply:
I would argue, that the purpose of the network client is the handling of the communication, and the given Sender classes should contain these constants as well (as the validation happens there). But if you would keep it here, I am ok with it, so this is not a blocker.

Reply:
I'll make this change in #187 so that i can update all usages in one place.